test/register_test.go

```diff
@@ -0,0 +1,62 @@
+package test
+
+import (
+	"encoding/json"
+	"github.com/gin-gonic/gin"
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"net/http/httptest"
+	"shortener-auth/database"
+	"shortener-auth/internal/common"
+	"shortener-auth/internal/common/http_actions"
+	"shortener-auth/internal/routing"
+	"strings"
+	"testing"
+)
+
+func TestRegister(t *testing.T) {
```

Comment:
where is 422, 401 tests?

Reply:
401 on public route?

Reply:
im about tests at all. 401 on login can be